mm2src/coins/hd_wallet/mod.rs

```diff
@@ -485,6 +485,63 @@ impl HDPathAccountToAddressId {
         Ok(account_der_path)
     }
 }
+/// Represents how a hierarchical deterministic (HD) address is selected.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(untagged)]
+pub enum HDAddressSelector {
+    /// Specifies the HD address using its structured account, chain, and address ID.
+    AddressId(HDPathAccountToAddressId),
+    /// Specifies the HD address directly using a BIP-44 compliant derivation path.
```

Comment:
It's not only BIP-44 as 84 and others are supported. I know this is my comment originally, but it's completely accurate.